opm/autodiff/ParallelDebugOutput.hpp

```diff
@@ -413,13 +400,9 @@ namespace Opm
                 // write all cell data registered in local state
                 for (const auto& pair : localCellData_) {
                     const auto& data = pair.second.data;
-                    const size_t stride = data.size()/localState_.numCells();
 
-                    for( size_t i=0; i<stride; ++i )
-                    {
-                        // write all data from local state to buffer
-                        write( buffer, localIndexMap_, data, i, stride );
-                    }
+                    // write all data from local data to buffer
+                    write( buffer, localIndexMap_, data);
```

Comment:
The data we write is `data::Solution` from opm-output where "stide" is not used. I think this is just some leftover from the time the reservoirState was written out directly.